core/domain/rights_manager.py

```diff
@@ -324,9 +324,12 @@ class Actor(object):
     def __init__(self, user_id):
         # Note that this may be None.
         self.user_id = user_id
+        self._is_admin = None
```

Comment:
done.
Yes you are right. Makes sense to do something similar with `is_moderator()`. Done!